Add FreeBSD warning message (#608)
jellyfin/jellyfin.org: docs/general/installation/index.mdx

The Jellyfin project and its contributors offer a number of pre-built binary packages to assist in getting Jellyfin up and running quickly on multiple systems.

:::caution

> [FreeBSD](https://www.freebsd.org/) and its derivatives, such as [TrueNAS CORE](https://www.truenas.com/truenas-core/), are **NOT** supported by Jellyfin and the .NET framework it uses.

Reviewer: ```suggestion
[FreeBSD](https://www.freebsd.org/) and its derivatives, such as [TrueNAS CORE](https://www.truenas.com/truenas-core/), are **NOT** supported by Jellyfin due to .NET officially not being compatible with these platforms.
```

> Even though there are builds available online for these platforms, they are unofficial and from [a separate project](https://github.com/Thefrank/jellyfin-server-freebsd). If you do encounter issues on these platforms, please ask for support in their respective support channels FIRST.

Reviewer: ```suggestion
Even though there are builds available online for these platforms, they are unofficial and from [a separate project](https://github.com/Thefrank/jellyfin-server-freebsd). If you do encounter issues on these platforms, please ask for support in their respective support channels first.
```

> [TrueNAS SCALE](https://www.truenas.com/truenas-scale/) is based on Linux and therefore is officially supported. Please install the Jellyfin app from its app repository.

Reviewer: ```suggestion
[TrueNAS SCALE](https://www.truenas.com/truenas-scale/) is based on Linux and therefore officially supported. Please install the Jellyfin app from its app repository. This app is not officially maintained by the Jellyfin team, therefore please use the TrueNAS support channels for help first.
```
